.travis.yml

```diff
@@ -1,13 +1,16 @@
 language: go
 sudo: false
+
+go:
+  - 1.6
+  - 1.7
+  - 1.8
+  - tip
+
 notifications:
   email: false
-go: 
- - 1.6.x
- - 1.7.x
- - 1.8.x
- - tip
+
 before_install:
-  - go get github.com/mattn/goveralls
+  - if [ "$TRAVIS_GO_VERSION" = "1.8" ]; then go get github.com/mattn/goveralls; fi
 script:
-  - $HOME/gopath/bin/goveralls -service=travis-ci
+  - if [ "$TRAVIS_GO_VERSION" = "1.8" ]; then $HOME/gopath/bin/goveralls -service=travis-ci; fi
```

Comment:
Don't really want to upload coverage results for every go version. There is a rate limit for coveralls which would be annoying if we hit it